content/advent-2015/symmetric-api-testing-in-go.md

```diff
@@ -0,0 +1,146 @@
++++
+author = ["Aditya Mukerjee"]
+date = "2015-12-26T00:00:00-05:00"
+title = "Symmetric API Testing"
+series = ["Advent 2015"]
+draft = true
```

Comment:
You can remove the `draft = true`. It's draft until merged so the pull request process handles that.